pkg/aspect/inventory.go

```diff
@@ -108,6 +33,8 @@ func Inventory() ManagerInventory {
 			NewMetricsManager(),
 		},
 
-		QuotaMethod: {},
+		QuotaMethod: {
+			NewQuotasManager(),
```

Comment:
should we add a TODO somewhere to remove the NewQuotasManager() from CheckMethod when the proxy is ready?